app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -303,251 +393,366 @@ class Contact extends React.Component {
     );
   };
 
+  renderContactFields(alternate) {
+    const prefix = alternate ? 'alternate_' : '';
+    return (
+      <Form>
+        <Form.Row>
+          <Form.Group as={Col} lg="12" controlId={`${prefix}contact_type`}>
+            <Form.Label className="input-label">CONTACT TYPE{schema?.fields[`${prefix}contact_type`]?._exclusive?.required && ' *'}</Form.Label>
```

Comment:
This should be "CONTACT RELATIONSHIP" now right?

Reply:
Yes, good catch: https://github.com/SaraAlert/SaraAlert/pull/1116/commits/61cf3ae89ad900096ea730617fc7e96b42de95ec